diff -ruN app0/Profile.xml app1/Profile.xml
--- app0/Profile.xml	2013-02-03 15:24:58.000000000 +0000
+++ app1/Profile.xml	2013-02-03 15:27:22.000000000 +0000
@@ -4,4 +4,5 @@
         <Auth username="root" password="123456"/>
     </SSH>
     <Ping url="socket://localhost:10022/"/>
+    <Ping url="socket://localhost:18080/"/>
 </Profile>
diff -ruN app0/Provisioning.xml app1/Provisioning.xml
--- app0/Provisioning.xml	2013-02-03 15:24:58.000000000 +0000
+++ app1/Provisioning.xml	2013-02-03 15:27:22.000000000 +0000
@@ -23,5 +23,6 @@
     <Target name="cleanup">
         <Exec>vboxmanage storageattach ${vbox.name} --storagectl "Floppy Controller" --port 0 --device 0 --medium none</Exec>
         <PortForward hostport="10022" guestport="22"/>
+        <PortForward hostport="18080" guestport="8080"/>
     </Target>
 </Provisioning>
diff -ruN app0/floppy0/ks.cfg app1/floppy0/ks.cfg
--- app0/floppy0/ks.cfg	2013-02-03 15:24:58.000000000 +0000
+++ app1/floppy0/ks.cfg	2013-02-03 15:27:22.000000000 +0000
@@ -4,7 +4,7 @@
 keyboard us
 network --onboot yes --bootproto dhcp --noipv6
 rootpw  --iscrypted $6$1EkdaY5asXuyqswK$gHoFAUdMMorGOLGdmZOIcoUunSedZJOCUhlR/88j/4JEIuy5Y9zLVriK6d5LBJpR7BY4yzI2jjeJbZg.7TXOX0
-firewall --service=ssh
+firewall --disabled
 authconfig --enableshadow --passalgo=sha512
 selinux --disabled
 timezone UTC
@@ -22,4 +22,4 @@

 %packages --nobase
 @core
-%end
\ No newline at end of file
+%end
diff -ruN app0/floppy0/post-install.sh app1/floppy0/post-install.sh
--- app0/floppy0/post-install.sh	2013-02-03 15:24:58.000000000 +0000
+++ app1/floppy0/post-install.sh	2013-02-03 15:19:20.000000000 +0000
@@ -7,4 +7,8 @@
 /media/cdrom/VBoxLinuxAdditions.run
 umount /dev/sr0
 rm -R /media/cdrom
+
+yum -y install tomcat6
+chkconfig tomcat6 on
+
 poweroff now
